### ISSUE TITLE
swagger:extension is ignored in API level

### ISSUE THREAD

**Issue by townewgokgok** [goadesign/goa#2271]
I specified a swagger extension according to [ReDoc vendor extensions](https://github.com/Redocly/redoc/blob/0eb1e66a10ad3add8db147e2bd6ce6ee333edd68/docs/redoc-vendor-extensions.md) like

```
var _ = API("...", func () {
    // ...
    Meta("swagger:extension:x-tagGroups", `[{"name":...}]`)
})
```
but it does not appear in `openapi.yaml`.

Shouldn't this `root.Meta` to be `root.API.Meta` ? 🤔 

https://github.com/goadesign/goa/blob/0e5c54ed0fd48ebc00de62a09090b542ee6ea27a/http/codegen/openapi/openapi_v2_builder.go#L51

**Comment by townewgokgok** [comment]
Oops, it seems unlikely that I can do what I want to do because the metadata output destination is in `info`, not the root.
~The problem is that the ReDoc vendor extension does not meet the OpenAPI specification (however, this is often the case and I am often in trouble).~
~I will try to deal with workarounds.~

It seems that vendor extensions are also allowed in the root.
https://github.com/OAI/OpenAPI-Specification/blob/597a638b50fe9848fff449fcad67290a839229be/schemas/v2.0/schema.json#L12-L16
Are you planning to add `Extensions` field to the type ` V2` ?

**Comment by townewgokgok** [comment]
It seems that @nitinmohan87 has changed this time.
https://github.com/goadesign/goa/commit/6d9200ca48bab5261f3929cef8f114cf7d55a7ec#diff-6ecbb03fe70b40b52aa6080b1153f04aL53

**Comment by townewgokgok** [comment]
Same here
https://github.com/goadesign/goa/commit/6d9200ca48bab5261f3929cef8f114cf7d55a7ec#diff-6ecbb03fe70b40b52aa6080b1153f04aL23

**Comment by townewgokgok** [comment]
Aside from the root level vendor extension, please check pull request #2273 to fix the original problem of this issue.
I'll create another issue about root level vendor extension later.

**Comment by townewgokgok** [comment]
This issue has been resolved by #2298.
